GPU/D3D11/TextureCacheD3D11.cpp

```diff
@@ -477,11 +477,12 @@ class TextureShaderApplierD3D11 {
 			const float right = u2 * invHalfWidth - 1.0f + xoff;
 			const float top = v1 * invHalfHeight - 1.0f + yoff;
 			const float bottom = v2 * invHalfHeight - 1.0f + yoff;
-			// Points are: BL, BR, TR, TL.
-			verts_[0].pos = Pos(left, bottom, -1.0f);
-			verts_[1].pos = Pos(right, bottom, -1.0f);
-			verts_[2].pos = Pos(right, top, -1.0f);
-			verts_[3].pos = Pos(left, top, -1.0f);
+			float z = 0.0f;  // was -1.0f for some reason
```

Comment:
Hmm, D3D9 also has -1.0f.  Pretty sure this was copied from GLES and was meant to be the minimum z, not that it probably matters.  It should change for D3D9 also, right?

Changing it in D3D9 has a positive impact to DBZ under D3D9, though it does not work correctly... seems familiar from GLES but I forget what it means is wrong...

-[Unknown]